### PR TITLE
Brs 1710 | Fixes error with popup cookie

### PR DESCRIPTION
**Pull request by LindaLawton**
# Code Quality
- [ ] Tests are written for the features.
- [ ] Reasonable logging have been added to the code.
- [x] I have compiled and tested my own code.

# Deployment
- [x] Moved Jira issue [BRS-1710](https://eg-xena.atlassian.net/browse/BRS-1710) to "Code complete".
- [ ] Issue [BRS-xxx](https://eg-xena.atlassian.net/browse/BRS-xxx) needs to be merged before merging this PR.
- [x] If accepted, reviewer can merge this PR.
- [x] This branch can be deleted after merge.

# Changes
- Fixes error with the plugin not properly saving the cookie
- Adds a single call to the xena api to show how to use the bearer token.

GDPR
======
Only fill out this part if the PR is a GDPR change. A PR is a GDPR change if:
* It touches any General personal data or Sensitivity personal data
* All changes to the Ids, Payroll (backend and frontend) and Partners (Xena)

__Always add 'GDPR' label to PR!__

Change type (select one):
- [ ] Normal Change
  - [ ]  Minor change
  - [ ]  significant change
  - [ ]  major change
- [x] Standard Change (Only minor changes)
- [ ] Emergency Change

More info on change types [here](https://github.com/EG-BRS/Documentation/wiki/Definition-of-GDPR-change)

Other:
- [ ] I have add the GDPR label to this PR.
- [ ] A risk analysis have been add to the Jira task.
- [ ] A test plan have been add to the Jira task.
- [ ] A post implementation test is needed. Write it on the jira.
- [x] All communication is done with encryption (https).
- [x] All Sensitivity personal data is encrypted in the DB. 
- [x] All viewing, searching, changesing and deletions of ['GDPR sensitive data'](https://github.com/EG-BRS/Documentation/blob/master/GDPR/Datatypes-examples.md) is logged.
- [ ] Will this PR added any new 'Sensitivity personal data' to our systems? __If YES, @thehusted must be added as reviewer.__

### CODE REVIEW

File: Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Startup.cs [hunk 1]
```diff
@@ -33,24 +34,25 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
-
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
+            services.AddXenaAPI(new XenaApiOptions() { XenaEndpoint = "https://test.xena.biz/api"});
```
Comment (goblinhero) [inline Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Startup.cs]: De her samples skal være offentlige? Er vi sikre på at xena test skal fremgå her?

File: Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Xena/XenaHttpClient.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        public async Task<string> GetAsyc(string accessToken, string apiEndPoint)
+        {
+           client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
+           return await client.GetStringAsync($"{_xenaEndpoint}{apiEndPoint}");
+        }
+
+        public static XenaHttpClient Init(XenaApiOptions options)
+        {
+            var response = new XenaHttpClient(options.XenaEndpoint);
+            return response;
```
Comment (goblinhero) [inline Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Xena/XenaHttpClient.cs]: To små detaljer: response => client og Init => Create.
Hvorfor har du både en factory method og en public constructor?

Comment (LindaLawton) [reply Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Xena/XenaHttpClient.cs]: I thought i would add more options but i dont want to over complicate things so removed it.

File: Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Startup.cs (hunk 1 above)
Comment (LindaLawton) [reply Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Startup.cs]: ... ok if you insist.

**Review by goblinhero — APPROVED** [review]
LGTM

**Pull request merged**